Jenkinsfile.performance.mainnet

```diff
@@ -0,0 +1,44 @@
+@Library('lisk-jenkins') _
+def echoBanner(msg) {
+	echo '----------------------------------------------------------------------'
+	echo msg
+	echo '----------------------------------------------------------------------'
+}
+
+def checkHttp(url) {
+	def result = sh script: "curl -s -f -o /dev/null ${url}", returnStatus: true
+	return (result == 0)
+}
+
+def waitForHttp(url) {
+	waitUntil { script { return (checkHttp(url) == true) } }
+}
+
+pipeline {
+	agent none
+	triggers {
+		// Triggers every week at midnight on Sunday morning
+		cron('0 0 * * 0')
```

Comment:
Assuming the deployment is performed at 12:00pm, the test can be delayed one hour or so. To make sure the process works stable and there are no pending background tasks.